### PR TITLE
Add Bit.Country testnet Tewai to testnet list

### PR DESCRIPTION
**Pull request by justinphamnz**
Added endpoints, logos and spec of Bit.Country Test Network
https://bit.country.
https://github.com/bit-country/Bit-Country-Blockchain.

Bit.Country Chain project is supported by Web3 Foundation. For details of the approval, please visit https://github.com/w3f/Grant-Milestone-Delivery/pull/59 

Thank you.

### CODE REVIEW

File: packages/apps-config/src/api/spec/bitcountry.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    start: 'BlockNumber',
+    end: 'Option<BlockNumber>',
+    price_per_block: 'Balance'
+  },
+  CountryId: 'u64',
+  CollectionId: 'u64',
+  CurrencyId: 'u32',
+  TokenId: 'u64',
+  CurrencyIdOf: 'CurrencyId',
+  BalanceIdOf: 'Balance '
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/bitcountry.ts:47]:
```suggestion
  BalanceIdOf: 'Balance'
```

Extra space.

**Review by jacogr — COMMENTED** [review]
Thank you. It looks good, one small comment.

**Comment by justinphamnz** [comment]
> Thank you. It looks good, one small comment.

Thanks @jacogr. I have applied the suggestion. 
Have a great day.

**Review by jacogr — APPROVED** [review]
Thank you!

**Pull request merged**